ui/packages/catalog-ui-search/src/main/webapp/component/hide-attribute/hide-attribute.view.js

```diff
@@ -28,6 +28,7 @@ function filterAndSort(attributes) {
       if (metacardDefinitions.metacardTypes[property]) {
         return !metacardDefinitions.metacardTypes[property].hidden
       } else {
+        // eslint-disable-next-line no-undef
```

Comment:
I think we should require the announcement component at the top. How does this work without the require?

Reply:
you're right, the call fails. fixed

Reply:
TODO there's a couple of these actually